src/java/org/apache/nutch/metadata/Nutch.java

```diff
@@ -80,4 +80,11 @@
 	public static final String STAT_PROGRESS = "progress";
 	/**Used by Nutch REST service */
 	public static final String CRAWL_ID_KEY = "storage.crawl.id";
+	
+	public static final String ARG_SEEDDIR = "url_dir";
+	public static final String ARG_CRAWLDB = "crawldb";
```

Comment:
As static finals above, it would be nice to augment the additions here with some Javadoc. They are public of course.